src/typescript/12-generics/generics.md

```diff
@@ -1,23 +1,23 @@
 @page learn-typescript/generics Generics
 @parent learn-typescript 12
+@outline 3
 
 @description Learn about what Generics are, why they are useful, and how to create a linked list using Generics in TypeScript.
 
 @body
 
-
 ## Overview
 
-In this part, we will:
+In this section, we will:
 
-- Understand the purpose and basics of generic functions
-- Understand how to make generic classes
-- Understand how to make recursive generic classes
-- Create a TreeNode recursive generic class
+- Explore the fundamentals and objectives of generic functions
+- Learn to construct generic classes
+- Delve into creating recursive generic classes
```

Comment:
These can be simplified too. Skip verbs like explore/learn/etc. and just say the thing, e.g. “Create recursive generic classes.”

Reply:
- Understand the fundamentals and objectives of generic functions
- Apply fundamentals to create generic classes
- Combine recursion with generic classes
- Create recursive generics

Is this better?